labgrid/driver/networkinterfacedriver.py

```diff
@@ -0,0 +1,85 @@
+# pylint: disable=no-member
+import attr
+
+from .common import Driver
+from ..factory import target_factory
+from ..step import step
+from ..util.agentwrapper import AgentWrapper
+from ..resource.remote import RemoteNetworkInterface
+
+
+@target_factory.reg_driver
+@attr.s(eq=False)
+class NetworkInterfaceDriver(Driver):
+    bindings = {
+        "iface": {"RemoteNetworkInterface", "USBNetworkInterface"},
+    }
+
+    def __attrs_post_init__(self):
+        super().__attrs_post_init__()
+        self.wrapper = None
+
+    def on_activate(self):
+        if isinstance(self.iface, RemoteNetworkInterface):
+            host = self.iface.host
+        else:
+            host = None
+        self.wrapper = AgentWrapper(host)
+        self.proxy = self.wrapper.load('network_interface')
```

Comment:
Maybe `agent` instead of `proxy`?

Reply:
This is a proxy object for the agent and we already use 'proxy' in the other drivers. So I'd tend to keep it for consistency.